### PR TITLE
Add empty state to parameters popover on Function expressions.

### PR DESCRIPTION
**Pull request by jomarko**
Screenshot_20230203_075925

### CODE REVIEW

File: packages/boxed-expression-component/src/expressions/FunctionExpression/ParametersPopover.tsx [hunk 1]
```diff
@@ -59,15 +61,31 @@ export const ParametersPopover: React.FunctionComponent<ParametersPopoverProps>
     });
   }, [setExpression]);
 
+  const addParameterButDoNotPropagatreEvent = useCallback(
```
Comment (yesamer) [inline packages/boxed-expression-component/src/expressions/FunctionExpression/ParametersPopover.tsx:64]: @jomarko I would unify `addParameterButDoNotPropagatreEvent` and `addParameter` methods and use it in both Buttons.

File: packages/boxed-expression-component/src/expressions/FunctionExpression/ParametersPopover.tsx [hunk 2]
```diff
@@ -29,6 +28,8 @@ import { useBoxedExpressionEditorI18n } from "../../i18n";
 import { useBoxedExpressionEditorDispatch } from "../BoxedExpressionEditor/BoxedExpressionEditorContext";
 import { DataTypeSelector } from "../ExpressionDefinitionHeaderMenu";
 import "./ParametersPopover.css";
+import { Button, EmptyState, EmptyStateIcon, Title } from "@patternfly/react-core";
+import { CubesIcon } from "@patternfly/react-icons";
```
Comment (jomarko) [inline packages/boxed-expression-component/src/expressions/FunctionExpression/ParametersPopover.tsx:32]: We should use most specific import as we did for 
`import { Button } from "@patternfly/react-core/dist/js/components/Button"`

**Review by jomarko — COMMENTED** [review]
Fix imports. Feedback by @tiagobento

**Comment by tiagobento** [comment]
It think it will look best if there's no distractions when we're displaying the EmptyState. No redundant buttons, no scroll bar...

File: packages/boxed-expression-component/src/expressions/FunctionExpression/ParametersPopover.tsx (hunk 1 above)
Comment (jomarko) [reply packages/boxed-expression-component/src/expressions/FunctionExpression/ParametersPopover.tsx]: done, please re-rereview the PR when you have a moment

**Comment by jomarko** [comment]
@tiagobento please re-review

**Review by yesamer — APPROVED** [review]

File: packages/boxed-expression-component/src/expressions/FunctionExpression/ParametersPopover.css [hunk 3]
```diff
@@ -30,6 +30,14 @@
   padding: 0 5px 5px 5px;
 }
 
+.boxed-expression-provider .parameters-editor-popover .parameters-editor .parameters-container-empty {
+  height: 200px;
+  overflow: hiden;
```
Comment (tiagobento) [inline packages/boxed-expression-component/src/expressions/FunctionExpression/ParametersPopover.css:35]: Typo.

**Review by tiagobento — CHANGES_REQUESTED** [review]
There's a typo on your CSS that might be a bug. Can you please provide a GIF/Screenshot of how it looks now?

File: packages/boxed-expression-component/src/expressions/FunctionExpression/ParametersPopover.css (hunk 3 above)
Comment (jomarko) [reply packages/boxed-expression-component/src/expressions/FunctionExpression/ParametersPopover.css]: done as eff511171f

**Comment by jomarko** [comment]
@tiagobento the current UX looks like

Screenshot_20230208_011816
Screenshot_20230208_011806

**Comment by tiagobento** [comment]
@jomarko I think there's a problem with the size of the Empty State icon... it looks too small..

**Review by tiagobento — APPROVED** [review]

**Pull request merged**